mygnupg.py

```diff
@@ -0,0 +1,64 @@
+import gnupg
+
+class mygnupg():
+    """
+    This class used for initial gnupg, 
+    create, import, and export key pairs.
+    Also, can encrypt and decrypt data.
+    """
+    def __init__(self):
+        """
+        The binary file should be different
+        in different os.
+        This is MacOS version.
+        """
+        self.gpg = gnupg.GPG(gnupghome='./gnupgkeys')
```

Comment:
The `gnupghome` argument should be passed a parameter from the constructor that has a default value. Also, it should probably use `pathlib` to remain OS agnostic. I realize the docstring says this is for MacOs but we probably shouldn't pin this class to any one OS if we can avoid it.

Reply:
Thanks, I will learn something about `pathlib` and path `gnupghome` as a parameter.